.circleci/config.yml

```diff
@@ -63,6 +63,7 @@ references:
     run:
       name: Run RSpec
       command: mkdir ~/rspec && cd decidim-$CIRCLE_JOB && bundle exec rake
+      no_output_timeout: 20m
```

Comment:
I don't think this is necessary with the latest change on the output format in specs
```suggestion
```